configure.ac

```diff
@@ -144,7 +144,7 @@ AC_HELP_STRING(
 esac],[enable_owr_java=no])
 AC_MSG_RESULT([$enable_owr_java])
 if test "x$enable_owr_java" = xyes; then
-  if test "x$found_introspection" != xyes; then
+  if test ! find ${prefix}/.. -name Owr-0.1.gir; then
```

Comment:
Is this going to result in a syntax error if a patch with a space is found, or multiple paths?